### ISSUE TITLE
ESLint parser not detected when listed in parserOptions.parser

### ISSUE THREAD

**Issue by GarrettGeorge** [depcheck/depcheck#352]
### Bug Description
ESLint parsers specified in `parserOptions.parser` are not read and listed under `Unused devDependencies`.

### Code snippets
Where the dependency is used:
See for details https://eslint.vuejs.org/user-guide/#usage
```
- "parser": "babel-eslint",
  "parserOptions": {
+     "parser": "babel-eslint",
      "sourceType": "module"
  }
```

Where the dependency is listed in package.json:

```
"devDependencies": {
    "babel-eslint": "^9.0.0"
  }
```

### Versions
 - `node -v`: `10.15.3`
 - `npm -v`: `6.4.1`
 - `depcheck --version`: `0.0.8`

### Extra info
I believe [this line](https://github.com/depcheck/depcheck/blob/master/src/special/eslint.js#L84) in `src/special/eslint.js` just needs to be change to include `parserOptions.parser`.

**Comment by 43081j** [comment]
Note that this is a vue-eslint-parser workaround, it isn't an option eslint supports.

It could still be added to depcheck but it would be solely for supporting the vue plugin, just FYI.

**Comment by rumpl** [comment]
not stale
